### PR TITLE
Update survey team for 2021 survey

### PR DESCRIPTION
**Pull request by nrc**
r? @rylev

### CODE REVIEW

**Review by Manishearth — DISMISSED** [review]

**Review by rylev — APPROVED** [review]
❤️

File: teams/community-survey.toml [hunk 1]
```diff
@@ -2,10 +2,9 @@ name = "community-survey"
 subteam-of = "community"
 
 [people]
-leads = ["nikomatsakis", "rylev"]
+leads = ["rylev"]
 members = [
-    "Manishearth",
-    "nikomatsakis",
```
Comment (rylev) [inline teams/community-survey.toml:8]: While this reflects the current reality, I'd like to keep @Manishearth and @nikomatsakis on the team if they still want to be.

Comment (rylev) [reply teams/community-survey.toml:8]: I see @Manishearth has approved the PR. Niko has been involved in this year's survey albeit in a much reduced role. We probably should keep him as a member.

Comment (nrc) [reply teams/community-survey.toml]: I have reinstated Niko :-)

**Review by rylev — APPROVED** [review]

**Comment by rylev** [comment]
I'm unable to merge this, so we'll need to wait for someone with permissions.

**Review by Manishearth — APPROVED** [review]
yeah, i might involve myself in the future again, but definitely not this year

**Pull request merged**